lighthouse-core/gather/gather-runner.js

```diff
@@ -101,7 +107,8 @@ class GatherRunner {
       .then(_ => driver.cacheNatives())
       .then(_ => resetStorage && driver.cleanAndDisableBrowserCaches())
       .then(_ => resetStorage && driver.clearDataForOrigin(options.url))
-      .then(_ => driver.blockUrlPatterns(options.flags.blockedUrlPatterns || []));
+      .then(_ => driver.blockUrlPatterns(options.flags.blockedUrlPatterns || []))
+      .then(_ => gathererResults.userAgent = [driver.getUserAgent()]);
```

Comment:
as much as the capitalization makes me cringe, this should be `gathererResults.UserAgent` to match the other names coming from gatherers